fluss-server/src/test/java/com/alibaba/fluss/server/coordinator/CoordinatorEventProcessorTest.java

```diff
@@ -612,12 +627,45 @@ void testCreateAndDropPartition() throws Exception {
                 nBuckets,
                 replicationFactor);
 
+        // mock CompletedSnapshotStore for partition1 and partition2
+        Set<TableBucket> tableBuckets4partition1 =
+                coordinatorContext.getAllBucketsForPartition(tableId, partition1Id);
+        Set<TableBucket> tableBuckets4partition2 =
+                coordinatorContext.getAllBucketsForPartition(tableId, partition2Id);
+        int sizeofCompletedSnapshotStore4partition1 = 0;
+        int sizeofCompletedSnapshotStore4partition2 = 0;
+        for (TableBucket tableBucket : tableBuckets4partition1) {
```

Comment:
nit:
We can just mock `completedSnapshotStore` for the partition1 which is to be drop to simlify the test